webapp/components/form/Dropdown/store/actions/useBlurInputValue.js

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react'
+
+import * as A from '@core/arena'
+
+import { State } from '../state'
+import { getItemsDialog } from './getItemsDialog'
+
+export const useBlurInputValue = ({ onChange }) =>
```

Comment:
can this be just `useBlurInput` or `useOnBlurInput` and called from outside with `onBlurInput`?